client/components/Container.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Button from './Button';
+
+const buttonText = 'SEARCH';
```

Comment:
I'm not sure you get much benefit from putting this into its own variable.

In a real-world app, you would be looking up the text from a mapping of internationalized text based on the user's preferred language. It would probably look something like
`text={intl.lookup('search')}`

Reply:
I'll research into this